region/lowernorfair/east.json

```diff
@@ -842,9 +854,28 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "canSwim" ],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                {"and": [
+                  "HiJump",
+                  "canSuitlessLavaDive",
```

Comment:
I think the lack of a tech in the reqs for the dive was indeed a mistake on my part. Are we comfortable with using the same tech as suitless lava though? A suitless acid dive has better physics than suitless lava but deals a lot more damage.

Reply:
It's the same ballpark. Seems ok.